tests/integration-tests.test.ts

```diff
@@ -22,7 +22,7 @@ describe('integration tests', () => {
    });
 
    const makeRequestEvent = (path: string, base?: RequestEvent, method?: string): RequestEvent => {
-      return _.extend(base || apiGatewayRequest(), { path: path, httpMethod: (method || 'GET') });
+      return _.extend(base, { path: path, httpMethod: (method || 'GET') });
```

Comment:
What happens if `makeRequestEvent` doesn't have a `base` provided? If we should always provide `base`, can we remove the optional flag?